README.md

```diff
@@ -189,7 +190,7 @@ To start using Summoner make sure that you have the following tools installed on
 
 We also have minimal version requirements for build tools:
 
-* [Cabal >= 2.0](https://www.haskell.org/cabal/)
+* [Cabal >= 2.2](https://www.haskell.org/cabal/)
```

Comment:
Should it be `2.4`?
```suggestion
* [Cabal >= 2.4](https://www.haskell.org/cabal/)
```

Reply:
Yeah, let's not overcomplicate things and put the one we are currently using :)